src/commcare_cloud/commands/inventory_lookup/inventory_lookup.py

```diff
@@ -67,6 +67,9 @@ def run(self, args, ssh_args):
         if ':' in address:
             address, port = address.split(':')
             ssh_args = ['-p', port] + ssh_args
+        if '@' not in address:
+            username = get_dev_username(args.env_name)
+            address = f'{username}@{address}'
```

Comment:
This means rather than using `ssh`'s built-in inference, cchq will use `COMMCARE_CLOUD_DEFAULT_USERNAME` to override that default if it's present, otherwise it will explicitly provide the same default (your current os username) that `ssh` would have come up with on its own?

Just trying to make sure my understanding of the intent/behavior is correct before I approve

Reply:
I think my main question is then what happens with anything dev users might currently have in their `~/.ssh/config`. I think I remember @millerdev having thoughts about this in a previous PR from some time ago, but I wasn't able to dig up where that was.

Reply:
Perhaps you were thinking of https://github.com/dimagi/commcare-cloud/issues/2410

Reply:
It is true that the ssh config can be used to overide the username provided you either:
1. know the specific addresses you want to override (or address patterns)
2. want to override it globally as suggested in #2410

I think that's fine to keep that, what about using using `ssh <host> -G | grep 'user '` to check the SSH username and provide a warning / error if it's not in the list of allowed hosts?